services/ui-src/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "module": "esnext",
     "baseUrl": "src",
     "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
+    "forceConsistentCasingInFileNames": false,
```

Comment:
Hopefully we don't actually need this change, maybe it was other changes that were merged while I was testing, but this workaround allowed me to resume my testing.

Reply:
Yeah I don't really know that we need this but not a huge deal for me.